rolecall/rolecall.py

```diff
@@ -52,7 +53,8 @@ def __init__(self, bot):
 
     def _record_entry(self, entry: Entry):
         """ record entry to settings file """
-
+        if type(entry.emoji) is discord.Emoji:
```

Comment:
please use `isinstance` instead
https://docs.quantifiedcode.com/python-anti-patterns/readability/do_not_compare_types_use_isinstance.html

Reply:
used `isinstance` as of 707ad4b